src/helpers/approval/approval-helper.js

```diff
@@ -11,4 +12,10 @@ export const loadWorkflowOptions = (filterValue = '') =>
 
 export const linkWorkflow = (id, resourceObject) => getWorkflowApi().linkWorkflow(id, resourceObject);
 export const unlinkWorkflow = (id, resourceObject) => getWorkflowApi().unlinkWorkflow(id, resourceObject);
-export const resolveWorkflows = (resourceObject) => getWorkflowApi().resolveWorkflows(resourceObject);
+
+export const listWorkflowsForObject = (resourceObject, limit = defaultSettings.limit, offset = defaultSettings.offset, filter = '') => {
+  const objectQuery = `app_name=${resourceObject.appName}&object_type=${resourceObject.objectType}&object_id=${resourceObject.objectId}`;
+  const paginationQuery = `&limit=${limit}&offset=${offset}`;
+  const filterQuery = `${filter.length > 0 ? `/?filter[name][contains]=${filter}` : ''}`;
```

Comment:
This condition is not required [contais_i]='' will behave as if no filter was passed.